misc/theme-cicada/src/opnsense/www/themes/cicada/assets/stylesheets/bootstrap-select/less/bootstrap-select.less

```diff
@@ -1,11 +1,27 @@
-@import "variables";
-
-span.form-control {
-  height: 34px !important;
-  padding: 6px 12px;
+/*!* {
```

Comment:
I fixed this... please don't bring it back :D

Reply:
I  need to tell René to fix that in his source then. So what exactly are you fixing, then I will fix it in my source and return that to René.

Reply:
I assume it will also apply to Theme-Tukan

Reply:
could be, see https://github.com/opnsense/plugins/commit/6b8ef75feca6efc433c40aa470f5e161335b709e